dev-init.sh

```diff
@@ -10,6 +10,8 @@ CLIENT_SECRET="obp-secret"
 
 
 # Start containers
+chmod +x ./env-prep/init/init-aws.sh
+ls -l ./env-prep/init/init-aws.sh
```

Comment:
I don't think these two lines are needed. The file's executable nature is persisted on git.

Reply:
I also tested this right now and it seems to work for me without these lines.

Reply:
well I had to do it, and also in the docs they mention that, so probably depends on `os`, or you start `make` command with `sudo` privileges  ?